### PR TITLE
Fix #87: Rename tables to be less connected with SSL topic

### LINKED ISSUE #87: Rename tables to be less connected with SSL topic
Currently, tables use the `ssl_*` prefix. We should rename them to use something more generic, for example, `mus_*`.

### PR DESCRIPTION
**Pull request by jandusil**
- Rename tables from ssl* prefix to mus*

### CODE REVIEW

File: docs/db/changelog/changesets/mobile-utility-server/1.5.x/20230227-initial-import.xml [hunk 1]
```diff
@@ -27,50 +27,50 @@
     <changeSet id="1" author="Petr Dvorak" logicalFilePath="mobile-utility-server/1.5.x/20230227-initial-import.xml">
         <preConditions onFail="MARK_RAN">
             <not>
-                <sequenceExists sequenceName="ssl_mobile_app_seq"/>
+                <sequenceExists sequenceName="mus_mobile_app_seq"/>
```
Comment (banterCZ) [inline docs/db/changelog/changesets/mobile-utility-server/1.5.x/20230227-initial-import.xml:30]: I am afraid that these changes have already been deployed to customers and should be implemented as incremental changes. @petrdvorak @zcgandcomp Am I correct?

Comment (zcgandcomp) [reply docs/db/changelog/changesets/mobile-utility-server/1.5.x/20230227-initial-import.xml:30]: Yes, and no. I believe we are fine from liquibase POV. BUT we need to do migration.

for me the sequence is as follows:
1) prepare migration scripts - I assume pure SQL for migration guide - part of the issue #87.
2) prepare the LB in new naming - from scratch installation will be fine and migration will be taken care of by the migration script.

i.e. whoever will want to update to 1.5.0 and has a previous installation has to run the migration script
and the rest is taken care of the liquibase.

We do not need to solve the cases when the liquibase itself is already executed somewhere as it's only testing of the snapshot.

File: docs/db/changelog/changesets/mobile-utility-server/1.5.x/20230227-initial-import.xml [hunk 2]
```diff
@@ -27,50 +27,50 @@
...
             </not>
         </preConditions>
         <comment>Create a new sequence for mobile app entities.</comment>
-        <createSequence sequenceName="ssl_mobile_app_seq" maxValue="9999999999999" cacheSize="20"/>
+        <createSequence sequenceName="mus_mobile_app_seq" maxValue="9999999999999" cacheSize="20"/>
```
Comment (zcgandcomp) [inline docs/db/changelog/changesets/mobile-utility-server/1.5.x/20230227-initial-import.xml:34]: no maxValues please.
startValue and incrementBy is required.

**Review by zcgandcomp — CHANGES_REQUESTED** [review]
update the proper attributes for sequences in LB.
Add migration script.

**Review by zcgandcomp — APPROVED** [review]
Looks OK

**Pull request merged**